apps/rpc/src/modules/workspace-sync/workspace-service.ts

```diff
@@ -167,6 +167,22 @@ export function getWorkspaceService(
     return `${local}@${domain}`
   }
 
+  function getLinkedWorkspaceId(localResolvable: User | Group | string): string | null {
```

Comment:
I think we should find a more descriptive name than localResolvable

Reply:
At this point a type alias would also be suitable. This union is repeated quite a lot